openadapt/config.py

```diff
@@ -13,12 +13,13 @@
     "DB_ECHO": False,
     "DB_FNAME": "openadapt.db",
     "OPENAI_API_KEY": "<set your api key in .env>",
-    #"OPENAI_MODEL_NAME": "gpt-4",
+    # "OPENAI_MODEL_NAME": "gpt-4",
     "OPENAI_MODEL_NAME": "gpt-3.5-turbo",
     # may incur significant performance penalty
     "RECORD_READ_ACTIVE_ELEMENT_STATE": False,
     # TODO: remove?
     "REPLAY_STRIP_ELEMENT_STATE": True,
+    "IGNORE_WARNINGS": True,
```

Comment:
@0dm what do you think about setting this to False by default?

Reply:
I think it's better to ignore warnings by default if we use the GUI, since the logger will just fill up with warnings. Maybe I could have it set IGNORE_WARNINGS to true within the app though.